robottelo/hosts.py

```diff
@@ -122,6 +122,7 @@ def __init__(self, hostname, auth=None, **kwargs):
 
         :param hostname: The fqdn of a ContentHost target
         :param auth: ('root', 'rootpass') or '/path/to/keyfile.rsa'
+        :param satellite: optional parameter satellite object.
```

Comment:
When backporting changes like this, its good to maintain the original commit via `cherry-pick`, in order to clearly separate it from other unique changes that you've committed for this branch.

Reply:
@mshriver  To make this PR smaller, I intentionally didn't include original commits via cherry-picks, so I just grabbed changes from `robottelo/hosts.py` to make this tests pass :)
Likewise, if you need related PRs that I referred to - #9633 #9662 #9362 #9310 and the rest are unique changes I've made for this branch.

Reply:
Thanks for the context, this helps. You have to be very careful when not chery-picking the commits, as you might be including function signature updates in `robottelo.hosts` classes without making the requisite updates that were included in the original commit(s).